### DISCUSSION TITLE
Off Screen Components

### DISCUSSION THREAD

**Discussion by drdan18** [livewire/livewire#8933] [1. Help]
I thought I saw this functionality but I am not sure how to search for it.  Maybe I am just making it up. 

I would like to turn off the rendering of a livewire component when it is scrolls off the screen.   I have a component that displays random quotes.  The height is variable as the quotes are different lengths.  It is find when reading the quotes but when you scroll down in the window and it pushes the quotes off screen it causes the page to jump up and down which is a little annoying.  

Is there a way to only re-render that component when the tag is visible?  

Thanks. 
Dan

**Comment by PhiloNL** [comment]
I don't think it's possible to have components render when scrolling but you can use something like Alpine to have a component fade-in for example using the intersect plugin https://alpinejs.dev/plugins/intersect.
